proto/internal/temporal/server/api/historyservice/v1/request_response.proto

```diff
@@ -638,6 +638,22 @@ message UpdateWorkflowExecutionResponse {
     temporal.api.workflowservice.v1.UpdateWorkflowExecutionResponse response = 1;
 }
 
+message ReadHistoryBranchRequest {
+    string namespace_id = 1;
+    int32 shard_id = 2;
+    bytes branch_token = 3;
+    int64 min_event_id = 4;
+    int64 max_event_id = 5;
```

Comment:
I think a quick comment on whether these are inclusive/exclusive here would be beneficial